language-support/ts/codegen/src/TsCodeGenMain.hs

```diff
@@ -625,8 +649,9 @@ writePackageJson packageDir sdkVersion (Scope scope) depends =
 --   ],
 --   ... perhaps other stuff ...
 -- }
+
 data PackageJson = PackageJson
-  { workspaces :: [T.Text]
+  { workspaces :: [Workspace]
```

Comment:
I'm not convinced that newtyping around record fields is really necessary.

Reply:
i undid this one.